lib/conduit_async_ssl.mli

```diff
@@ -33,7 +33,7 @@ val ssl_connect :
   ?verify:(Ssl.Connection.t -> bool Deferred.t) ->
   Reader.t ->
   Writer.t ->
-  (Reader.t * Writer.t) Deferred.t
+  (Reader.t * Writer.t) Deferred.Or_error.t
```

Comment:
On 01/06/2016 21:45, Rudi Grinberg wrote:
> Out of curiousity, is it possible to just use Conduit_async for your use
> case?

Absolutely, I actually changed my code to use |Conduit_async| :)